pkg/local/sorted_queue.go

```diff
@@ -0,0 +1,28 @@
+package local
+
+// A SortedQueue implements heap.Interface and holds strings.
+type SortedQueue []string
```

Comment:
It's a PriorityQueue :-/  Let's rename after everything else is done, otherwise GitHub becomes a mess.

Reply:
Modified the type name, I'll modify the file name when we're finished